components/plurals/src/lib.rs

```diff
@@ -191,31 +191,33 @@ pub enum PluralCategory {
 impl PluralCategory {
     /// Returns an ordered iterator over variants of [`Plural Categories`].
     ///
+    /// Categories are returned in alphabetical order.
+    ///
```

Comment:
`opinion`: as you recognized in your PR comment, there is a discussion about the order, and in particular I am in a bad position to review this change since I expressed a strong opinion there.

I want to stress that my opinion there is non-blocking and if the group will decide to go against my recommendation I will accept it and thus do not consider it blocking for this PR. Just a change that I disagree with to match a proposal that I disagree with.

Reply:
I vote to land this as-is and update it to reflect the decision in ECMA-402.  I added https://github.com/tc39/ecma402/issues/578 to the agenda for the July 1 meeting.